test/standalone/server.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import express from 'express';
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import uuidv4 from 'uuid/v4';
+import { resolve } from 'path';
+import Sample from './app';
+import { resetServerContext } from '../../src';
+
+const getNonce = () => Buffer.from(uuidv4()).toString('base64');
```

Comment:
see other comment about just using a simple alternative for this one rather than adding a new dependency

Reply:
Fixed